tests/05_02_01-last_fork_active_powpeg_redeem_script.js

```diff
@@ -0,0 +1,87 @@
+const chai = require('chai');
+chai.use(require('chai-as-promised'));
+const expect = chai.expect;
+const peglib = require('peglib');
+const bitcoin = peglib.bitcoin;
+const {compareFederateKeys} = require('../lib/federation-utils');
+const {getRskTransactionHelpers} = require('../lib/rsk-tx-helper-provider');
+const redeemScriptParser = require('@rsksmart/powpeg-redeemscript-parser');
+const CustomError = require('../lib/CustomError');
+const removePrefix0x = require('../lib/utils').removePrefix0x;
+const {getBridge, getLatestActiveForkName} = require('../lib/precompiled-abi-forks-util');
+
+// in order to run this as a single test file, it requires a federation change so follow the following command
+// npm run run-single-test-file 04_00_02-fedchange.js,05_02_01-last_fork_active_powpeg_redeem_script.js
```

Comment:
Since we are not using the `fulfillRequirementsToRunAsSingleTestFile` function here, I don't think this comment applies. You can remove it.

Reply:
Actually it does, since the fulfillRequirementsToRunAsSingleTestFile is done in 04_00_02-fedchange.js that not only set the fork activation but does the fed change.
The comment is to run the file as single, but the other as precondition.